language/pattern_matching_spec.rb

```diff
@@ -1136,6 +1173,16 @@ def ===(obj)
       end
     end
 
+    ruby_version_is ""..."2.7" do
+      it "raises NoMatchingPatternError if no pattern matches for one-line patterns" do
+        -> {
+          eval <<~RUBY
+            0 in 1
+          RUBY
+        }.should raise_error(NoMatchingPatternError, /\[0, 1\]/)
+      end
+    end
```

Comment:
This seems far from the other changes, and yet it is related.
I think we should have a separate top-level `describe` for `Single-line pattern matching`.

I think this spec never runs given the `ruby_version_is "2.7" do` on top and this meaning < 2.7 (there is no pattern matching before 2.7)